e2e_test.go

```diff
@@ -14,6 +14,8 @@
 
 package alloydbconn_test
 
+// [START alloydb_databasesql_connect_connector]
```

Comment:
do we want all the imports to be shown for both samples?

Reply:
Fixed this by splitting into two files.